h5bp/tls/policy_balanced.conf

```diff
@@ -1,13 +1,9 @@
 # ----------------------------------------------------------------------
-# | SSL policy - Intermediate                                          |
+# | SSL policy - Balanced                                              |
 # ----------------------------------------------------------------------
 
-# For services that don't need backward compatibility, the parameters below
-# provide a higher level of security.
-#
-# (!) This policy enforces a mildly strong SSL configuration, which may raise
-#     errors with old clients.
-#     If a more compatible profile is required, use the "deprecated" policy.
+# For services that need to support a wide range of clients, this configuration
+# is raisonnably balanced.
```

Comment:
reasonably

Reply:
Oh good catch, thanks!

Reply:
> reasonably

This is about you and me, doing our best, to keep Huell happy.